contracts/staking/ComplexRewardPerSec.sol

```diff
@@ -144,7 +144,7 @@ contract ComplexRewarderPerSec is IComplexRewarder, Ownable, ReentrancyGuard {
     /// @notice Function called by FarmDistributor whenever staker claims VELA harvest. Allows staker to also receive a 2nd reward token.
     /// @param _user Address of user
     /// @param _amount Number of LP tokens the user has
-    function onVelaReward(uint256 _pid, address _user, uint256 _amount) external override onlyDistributor nonReentrant {
+    function onVelaReward(uint256 _pid, address _user, uint256 _amount) external override onlyDistributor {
```

Comment:
Why remove nonReentrant?

Reply:
Because we are calling onVelaReward from TokenFarm nonReentrant function(deposit and withdraw).
We dont need to duplicate.